Scripts/Services/Town Cryer/Quests/AVisitToCastleBlackthornQuest.cs

```diff
@@ -0,0 +1,82 @@
+using Server;
+using System;
+using Server.Items;
+using Server.Mobiles;
+using Server.Services.TownCryer;
+
+namespace Server.Engines.Quests
+{
+    public class AVisitToCastleBlackthornQuest : BaseQuest
+    {
+        private object _Title = 1158197;
+
+        /* A Visit to Castle Blackthorn */
+        public override object Title { get { return _Title; } }
+
+        /*It seems that Castle Blackthorn has some secrets that are worth investigating. Your history on how Blackthorn even became 
+         * king is a little fuzzy. You decide a visit to Castle Blackthorn would be worthwhile.*/
+        public override object Description { get { return 1158198; } }
+
+        /* You decide against accepting the quest. */
+        public override object Refuse { get { return 1158130; } }
+
+        /* Visit Castle Blackthorn in Northern Britain. */
+        public override object Uncomplete { get { return 1158199; } }
+
+        /* You have braved the wilds of Britannia and slayed a mighty beast! You have meticulously documented your kill 
+         * and submitted it to the Ranger's Guild for evaluation. If luck was on your side you may indeed have 
```

Comment:
This message seems to be for the Huntmaster quest and not blackthorn.

Reply:
The only quest i was able to complete was the huntsmaster, and i got a deed. ALso, that cliloc is correct, just the wrong discription which doesnt matter but ill update it.